samples/azure-quantum/grover/Grover.ipynb

```diff
@@ -0,0 +1,710 @@
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%simulate SearchForMarkedInput nQubits=3 idxMarked=6"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "## Running Grover's algorithm in Azure Quantum\n",
+    "To run the operation on the Azure Quantum service, you must first connect to your workspace by running `%azure.connect` with the workspace's resource ID and location."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%azure.connect \"\" location=\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "Then use `%azure.target` to select the target you'd like to use for job submission."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Loading package Microsoft.Quantum.Providers.IonQ and dependencies...\n",
+      "Active target is now ionq.simulator\n"
+     ]
+    },
+    {
+     "data": {
+      "application/x-qsharp-data": "{\"id\":\"ionq.simulator\",\"current_availability\":{},\"average_queue_time\":1}",
+      "text/html": [
+       "<table><thead><tr><th style=\"text-align: start;\">Target ID</th><th style=\"text-align: start;\">Current Availability</th><th style=\"text-align: start;\">Average Queue Time (Seconds)</th></tr></thead><tbody><tr><td style=\"text-align: start;\">ionq.simulator</td><td style=\"text-align: start;\">Available</td><td style=\"text-align: start;\">1</td></tr></tbody></table>"
+      ],
+      "text/plain": [
+       "Target ID      Current Availability Average Queue Time (Seconds)\n",
+       "-------------- --------- -\n",
+       "ionq.simulator Available 1\n"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%azure.target ionq.simulator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "To submit the job, use `%azure.submit` with the Q# operation name and any required parameters. This will return immediately after the job is created. Alternatively, you can use %azure.execute, which will submit the job and wait for it to complete."
```

Comment:
Nit, but `%azure.execute` should be typeset the same way as other magic commands, namely inside `` ` `` marks. It may also be good to link to the IQ# magic command reference the first time each command is mentioned?

Reply:
Thank you for catching the formatting problem! I fixed that and added links to the docs each time a magic command is mentioned.